com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/HDShaderGUI.cs

```diff
@@ -93,6 +121,10 @@ protected static void ResetMaterialCustomRenderQueue(Material material)
             kUseSplitLighting, kTransparentBackfaceEnable
         };
 
+        /// <summary>
+        /// For ShaderGraph Only, synchronize a set of properties that is needed for ShaderGraph materials to work correctly.
```

Comment:
Synchronize a set of properties that Unity requires for Shader Graph materials to work correctly. This function is for Shader Graph only.